site/components/Wallet.js

```diff
@@ -18,7 +19,25 @@ const getLastConnector = () =>
 const removeLastConnector = () =>
   window.localStorage.removeItem(persistLastConnectorKey)
 
-const Wallet = function () {
+const WalletButton = function ({ cta = false, onClick }) {
```

Comment:
We have a default value for `cta` defined here and below in `Wallet` but we should only have it set in a single place (preferably here as it is "deeper" in the components hierarchy).

Anyway, the default could be change to `true` as that is the most common use case.

Reply:
I believe `false` is the most common use case here, because it's the default behavior of the original **Connect Wallet** button:

<img width="827" alt="Captura de Tela 2024-12-05 às 11 16 30" src="https://github.com/user-attachments/assets/54bb395e-9feb-4749-b7cc-372a2853098e">